.ci/magician/provider/version.go

```diff
@@ -23,6 +23,18 @@ func (v Version) String() string {
 	return "unknown"
 }
 
+func (v Version) ProviderName() string {
+	switch v {
+	case GA:
+		return "google"
+	case Beta:
+		return "google-beta"
+	case Alpha:
```

Comment:
If this is the version of the provider, I'd go with "Private". I think alpha tends to be used for the API.